version/version.go

```diff
@@ -10,11 +10,11 @@ var (
 	// compiler.
 	GitCommit string
 
-	// The main version number that is being run at the moment.
+	// The next version number that will be released in this series.
 	//
 	// Version must conform to the format expected by github.com/hashicorp/go-version
 	// for tests to work.
-	Version = "1.11.3"
+	Version = "1.11.7"
```

Comment:
Since the current version of consul on `release/1.11.x` is `1.11.6`, this has been bumped to the NEXT version, `1.11.7`. After `1.11.7` is released, this version should be bumped to `1.11.8` if there will be another patch in this series.